invenio_app_ils/mappings/v6/patrons/patron-v1.0.0.json

```diff
@@ -0,0 +1,19 @@
+{
+  "mappings": {
+    "patron-v1.0.0": {
+      "date_detection": false,
+      "numeric_detection": false,
+      "properties": {
+        "id": {
+          "type": "integer"
+        },
+        "name": {
+          "type": "text"
+        },
+        "email": {
+          "type": "text"
```

Comment:
it should be keyword in ES

Reply:
Why to be so strict? I mean partial search on emails could happen no?